### PR TITLE
Update mash sketch version 2.3

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/mash/mash_sketch.xml [hunk 1]
```diff
@@ -55,29 +62,104 @@
                     </when>
                 </conditional>
                 <param type="integer" name="minimum_kmer_copies" argument="-m" value="1" min="1" max="1000" label="Minimum copies of each k-mer required to pass noise filter"/>
+                <param type="integer" name="target_coverage" argument="-c" value="" min="0" max="500" optional="true" label="Target coverage" help="If specified, sketching will conclude if this coverage is reached before the end of the input file (estimated by average k-mer multiplicity)"/>
+                <param type="integer" name="genome_size" argument="-g" value="" min="1000" max="15000000" optional="true" label="Genome size" help="If specified, will be used for p-value calculation instead of an estimated size from k-mer content"/>
```
Comment (bgruening) [inline tools/mash/mash_sketch.xml:66]: 15M seems to be too small, isn't it?

**Review by bgruening — COMMENTED** [review]
Very nice! Just one small comment.

File: tools/mash/mash_sketch.xml (hunk 1 above)
Comment (gallardoalba) [reply tools/mash/mash_sketch.xml:66]: True, I used the biggest bacterial genome.

File: tools/mash/mash_sketch.xml [hunk 2]
```diff
@@ -63,7 +63,7 @@
                 </conditional>
                 <param type="integer" name="minimum_kmer_copies" argument="-m" value="1" min="1" max="1000" label="Minimum copies of each k-mer required to pass noise filter"/>
                 <param type="integer" name="target_coverage" argument="-c" value="" min="0" max="500" optional="true" label="Target coverage" help="If specified, sketching will conclude if this coverage is reached before the end of the input file (estimated by average k-mer multiplicity)"/>
-                <param type="integer" name="genome_size" argument="-g" value="" min="1000" max="100000000000" optional="true" label="Genome size" help="If specified, will be used for p-value calculation instead of an estimated size from k-mer content"/>
+                <param type="integer" name="genome_size" argument="-g" value="" min="1000" max="10  0000000000" optional="true" label="Genome size" help="If specified, will be used for p-value calculation instead of an estimated size from k-mer content"/>
```
Comment (bgruening) [inline tools/mash/mash_sketch.xml:66]: is there a space included? maybe remove max?

**Review by bgruening — APPROVED** [review]

**Pull request merged**